src/test/java/core/basesyntax/ValidationConstant.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public enum ValidationConstant {
```

Comment:
Using enum for saving property values looks not corresponding to clean code suggestions. It is better to use usuals  constants inside of this class. This make code simpler and you wouldn't need to return the Object in getValue() method with additional manual casting later in the code.